python/sparkdl/transformers/tf_image.py

```diff
@@ -236,6 +239,6 @@ def _convertOutputToVector(self, df, tfs_output_col):
         Converts the output python list to MLlib Vector.
         """
         return (
-            df.withColumn(self.getOutputCol(), list_to_vector_udf(df[tfs_output_col]))
+            df.withColumn(self.getOutputCol(), JVMAPI.list_to_vector(df[tfs_output_col]))
```

Comment:
as mentioned in #15 , let's change that so that the python list is handlded directly by py4j.